From b73d343bd2bd069c8f57aad785bb84e29ffe5f01 Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Fri, 12 Oct 2018 16:33:04 +0800
Subject: [PATCH 4860/5242] MLK-19911 ARM64: dts: imx8qm: fix PCIEA rsrc id
 for smmu programming

commit  dfd08014d12358fa71e8aaa145c72c11d0d72ede from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix PCIEA rsrc id, then uboot could program SID correctly for PCIEA.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Reviewed-by: Andy Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8qm-mek-dom0-dpu2.dts     |    1 +
 .../boot/dts/freescale/fsl-imx8qm-mek-dom0.dts     |    1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0-dpu2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0-dpu2.dts
index 4810f4b..2ab7970 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0-dpu2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0-dpu2.dts
@@ -531,6 +531,7 @@
        #stream-id-cells = <1>;
        iommus = <&smmu>;
        xen,passthrough;
+       fsl,sc_rsrc_id = <SC_R_PCIE_A>;
 };
 
 &pcieb {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
index fcd470e..33ac207 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
@@ -527,6 +527,7 @@
        #stream-id-cells = <1>;
        iommus = <&smmu>;
        xen,passthrough;
+       fsl,sc_rsrc_id = <SC_R_PCIE_A>;
 };
 
 &pcieb {
-- 
1.7.9.5

